templates/common/.devcontainer/Dockerfile/postgresql/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG VARIANT=bullseye
+FROM --platform=amd64 mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+RUN export DEBIAN_FRONTEND=noninteractive \
+     && apt-get update && apt-get install -y xdg-utils \
+     && apt-get update && apt-get install --yes --no-install-recommends postgresql-client \
```

Comment:
I'm not sure we want to have a Dockerfile for a db client.  I'd rather pivot on lang / java instead of a db choice.